src/components/Search/Search.tsx

```diff
@@ -25,7 +25,7 @@ export type handleClickSearch = (
   event: KeyboardEvent<HTMLInputElement>,
   { suggestionValue, method }: { suggestionValue: object[]; method: string }
 ) => void;
-export type handleFetchPackages = ({ value: string }) => Promise<void>;
+export type handleFetchPackages = (arg: { value: string }) => Promise<void>;
```

Comment:
Please, why do you find "arg:" necessary? I believe to be irrelevant

Reply:
Haha this one was tricky, I didn't encounter this before 😉 
but that particular syntax didn't work as intended.
tldr: It only **renamed** `value` to `string`.
It didn't say that `value` is of type `string`.
see https://github.com/microsoft/TypeScript/issues/9657

To interpret `{ value: string }` as a type like intended and not a rename in a destructure assignment (which is apparently possible) you need to add the argument name.

Reply:
Very interesting! but in this case i would then do: `({value}: { value: string})`. It's more readable IMO, otherwise people can get confused